odoo/http.py

```diff
@@ -1530,6 +1529,35 @@ def db_monodb(httprequest=None):
         return dbs[0]
     return None
 
+def module_boot(db=None):
+    server_wide_modules = odoo.conf.server_wide_modules or []
+    serverside = ['base', 'web']
+    dbside = []
+    for i in server_wide_modules:
+        if i in addons_manifest and i not in serverside:
+            serverside.append(i)
+    monodb = db or db_monodb()
+    if monodb:
+        dbside = module_installed_bypass_session(monodb)
+        dbside = [i for i in dbside if i not in serverside]
+    addons = serverside + dbside
+    return addons
+
+def module_installed(environment):
+    # Candidates module the current heuristic is the /static dir
+    # Retrieve database installed modules
+    return environment['ir.module.module']._installed_sorted()
+
+def module_installed_bypass_session(dbname):
+    try:
+        registry = odoo.registry(dbname)
+        with registry.cursor() as cr:
+            return module_installed(
+                environment=odoo.api.Environment(cr, odoo.SUPERUSER_ID, {}))
+    except Exception:
+        pass
+    return {}
```

Comment:
doesn't that stuff belong somewhere in `odoo.modules` rather than `odoo.http`, which is concerned about dispatching requests only?

perhaps `get_modules_order` shouldn't be injected from here

Reply:
@odony 
done, although admittedly it's a bit artificial. Another solution would be to inject `get_modules_order` in the qcontext each time it is necessary (3 times in web's controller ; 3 more times in pos' controller), which is doable, of course.

There is friction between modules.module and http.py, I think it has been revealed by this PR, but was preexistent
The `addons_manifest` global variable is an example of this....

Reply:
woops, did not read the follwing comment